htdocs/compta/stats/casoc.php

```diff
@@ -250,8 +265,10 @@
 
 // Show Array
 $catotal = 0;
+$catotal_ht = 0;
+
 if ($modecompta == 'CREANCES-DETTES') {
-	$sql = "SELECT DISTINCT s.rowid as socid, s.nom as name, s.zip, s.town, s.fk_pays,";
+	$sql = "SELECT DISTINCT s.rowid as socid, s.nom as name, s.name_alias, s.zip, s.town, s.fk_pays,";
```

Comment:
If you add a field into the select, you must also add it into the group by

Reply:
Done @Eldy, thanks for your review